src/app/design/frontend/base/default/template/payment/form/omise/omiseoffsiteinstalment.phtml

```diff
@@ -0,0 +1,33 @@
+<?php
+$code = $this->getMethodCode();
+$backends = $this->getInstalmentBackends();
+?>
+
+<ul id="payment_form_<?php echo $code; ?>" class="banks-list form-list" style="display:none;">
+
+    <?php if (count($backends)) { foreach ($backends as $backend) { ?>
+        
+    <li class="item">
+        <input class="validate-one-required-by-name" id="<?=$backend->_id;?>" type="radio" name="payment[type]" value="<?=$backend->_id;?>" />
+        <label for="<?=$backend->_id;?>">
+            <div class="omise-logo-wrapper <?=$backend->_bankcode;?>">
+                <div class="logo-<?=$backend->_bankcode;?>"></div>
+            </div>
+            <div class="omise-banking-text-wrapper">
+                <span class="title"><?=$this->__($backend->_bankname);?></span><br/>
+                <select name="payment[terms_<?=$backend->_id;?>]" id="payment_term_<?=$backend->_id;?>">
+                    <?php foreach ($backend->allowed_installment_terms as $termMonths) { ?>
+                        <option value="<?=$termMonths;?>"><?=$termMonths;?> <?=$this->__('months');?></option>
+                    <?php } ?>
+                </select>
+            </div>
+        </label>
+    </li>
+
+    <?php }} else { ?>
```

Comment:
@jonrandy this line looks strange, I like WordPress way when it comes to having PHP code in html file, something like:
```html
<?php if (count($backends)) : ?>
    <?php foreach ($backends as $backend) : ?>
        <li class="item">
            <!-- ... -->
        </li>
    <?php endforeach; ?>
<?php else : ?>
    <li class="item">
        <?=$this->__('There are no instalment plans available for this purchase amount.');?>
    </li>
<?php endif; ?>
```
It is more easy to read as usually HTML code inside PHP condition-block tends to be long and is easy to get lost while reading over the lines.

Reply:
Ugh - `<?php` everywhere. Unnecessary. Harder to read

Reply:
And bizarre `:`'s

Reply:
Yup it's unnecessary, but it has more meaning when you've lost from scrolling down a couple lines of html code.
Out of topic, maybe at least having an indent between them would be more better than having everything at the same alignment without an indent.

cc @jacstn just need a third opinion about this.

Reply:
@guzzilar @jonrandy when you have html block you should use 
`<?php if ($condition) : ?>`
`<div></div>`
`<?php endif; ?>`

that is purpose `PHP` invented that to embed `if/for/foreach/while` statements in HTML code, It looks way more elegant and pretty..
however Jonnys version is not error. There was only invented other form for this particular case.